drupal/settings/production.settings.php

```diff
@@ -41,3 +41,6 @@
 
 // Disable temporary file deletion (GOVCMSD8-576).
 $config['system.file']['temporary_maximum_age'] = 0;
+if (is_numeric($image_gc = getenv('GOVCMS_FILE_TEMP_MAX_AGE'))) {
+  $config['system.file']['temporary_maximum_age'] = $image_gc;
+}
```

Comment:
GC doesn't only apply to images:

```suggestion
if (is_numeric($file_gc = getenv('GOVCMS_FILE_TEMP_MAX_AGE'))) {
  $config['system.file']['temporary_maximum_age'] = $file_gc;
}
```